pkg/backend/snapshot_test.go

```diff
@@ -401,6 +397,27 @@ func TestSamesWithOtherMeaningfulChanges(t *testing.T) {
 		assert.NoError(t, err)
 	}
 
+	// Source position is not a meaningful change, and we batch them up for performance reasons
+	manager, sp := MockSetup(t, snap)
+	sourceUpdated := NewResource(resourceA.URN)
```

Comment:
Should this be setting `sourceUpdated.SourcePosition`?

Reply:
It should, it was at some point and I must have deleted it when moving it around. Will re-add.